transports/websocket-websys/src/lib.rs

```diff
@@ -59,20 +83,20 @@ use crate::web_context::WebContext;
 ///     .multiplex(yamux::Config::default())
 ///     .boxed();
 /// ```
-///
 #[derive(Default)]
 pub struct Transport {
     _private: (),
 }
 
-/// Arbitrary, maximum amount we are willing to buffer before we throttle our user.
+/// Arbitrary, maximum amount we are willing to buffer before we throttle our
+/// user.
 const MAX_BUFFER: usize = 1024 * 1024;
 
 impl libp2p_core::Transport for Transport {
-    type Output = Connection;
+    type Dial = Pin<Box<dyn Future<Output = Result<Self::Output, Self::Error>> + Send>>;
     type Error = Error;
     type ListenerUpgrade = Ready<Result<Self::Output, Self::Error>>;
-    type Dial = Pin<Box<dyn Future<Output = Result<Self::Output, Self::Error>> + Send>>;
+    type Output = Connection;
```

Comment:
Please avoid unnecessary reformatting.

Reply:
fixed in [289b0d6](https://github.com/libp2p/rust-libp2p/pull/5025/commits/289b0d6dac0aa29c7c81fab8b849f1731a4d021b)